lib/composers/untilComposer.js

```diff
@@ -1,20 +1,19 @@
 const {makeModifierComposer} = require('../utils/makeComposer');
 const composers = require('../constants/composer');
 const {validate, validators} = require('../validation');
-const {unregisterLeaf} = require('../utils/unusedExpressionWatchers');
 const SuitestError = require('../utils/SuitestError');
 const texts = require('../texts');
 
 /**
  * Until composer populates chain with .until() method,
  * which accepts another chain and stores it under 'until' property
  */
-const untilComposer = makeModifierComposer(composers.UNTIL, ['until'], (data, chain) => {
+const untilComposer = makeModifierComposer(composers.UNTIL, ['until'], (classInstance, data, chain) => {
```

Comment:
classInstance only used in this place?

Reply:
used mainly for expression watchers and in then chain to access websockets